docs_src/environment_variables.md

```diff
@@ -0,0 +1,20 @@
+When running docker-run.sh script, we support environment variables as input for containers. Here is a list of environment variables and how you can apply it to docker-run.sh script as input
+
+| Environment Variable   | Purpose                                                                 |
+| -----------------------| ------------------------------------------------------------------------|
+| STREAM_DENSITY_MODE=1  | for starting single container stream density testing                    |
+| RENDER_MODE=1          | for displaying pipeline and overlay CV metadata                         |
+| LOW_POWER=1            | for using GPU usage only based pipeline for Core platforms              |
```

Comment:
not sure this statement is true: is it only for Core platforms?

Reply:
I'm not sure neither, these descriptions are from get-options.sh existed in the repo. Maybe @brian-intel will know better.

Reply:
hmm. maybe Xeon platform to have this too?

Reply:
This was true for the DLStreamer pipelines. I think this will be reworked and possibly replaced with the performance_mode parameter https://github.com/intel-retail/automated-self-checkout/blob/main/benchmark-scripts/benchmark.sh#L16

Reply:
are we resolving this yet? not totally sure

Reply:
@brian-intel should we create an issue to rework and replaced with performance_mode?

Reply:
We should keep it as it is for now since the LOW_POWER setting isn't currently used in the dlstreamer pipelines

Reply:
Hi @brian-intel
 the LOW_POWER env is being used when i search the code in multiple places and it seems to applies to both use cases... 
so i am not sure about the case that not being used in dlstreamer pipelines.... 
(**run_gst.sh** has this and also **yolov5s.sh underneath the dlstreamer/ folde**r has this too)

Reply:
after discussion, I have updated LOW_POWER description, @jim-wang-intel